### ISSUE TITLE
Azure org output of supported features not working

### ISSUE THREAD

**Issue by sgordon46** [PaloAltoNetworks/terraform-provider-prismacloud#269]
## Describe the bug

When following the documentation in trying to report back on the supported features for an azure org, I'm receiving an empty set

## Expected behavior

I would expect to see this similar output 

```features_supported_azure_account = toset([
  "Agentless Scanning",
  "Auto Protect",
  "Remediation",
  "Serverless Function Scanning",
])
```

## Current behavior

This is the current output

```
features_supported_azure_tenant = toset([])
```

## Steps to reproduce

running the following terraform deployment

```
terraform {
  required_providers {
    prismacloud = {
      source = "PaloAltoNetworks/prismacloud"
      version = "1.5.0"
    }
  }
}

provider "prismacloud" {
    json_config_file = "../.prismacloud_auth.json"
    
}

data "prismacloud_account_supported_features" "azure_account" {
  cloud_type      = "azure"
  account_type    = "account"
  deployment_type = "azure"
}

output "features_supported_azure_account" {
    value = data.prismacloud_account_supported_features.azure_account.supported_features
}

 data "prismacloud_account_supported_features" "azure_tenant" {
   cloud_type      = "azure"
  account_type    = "tenant"
    deployment_type = "azure"
}

output "features_supported_azure_tenant" {
    value = data.prismacloud_account_supported_features.azure_tenant.supported_features
}
```

**Comment by sgordon46** [comment]
Should be fixed with #270

**Comment by AnushreeHS** [comment]
@sgordon46 The documentation has been updated for better clarity in the onboarding flow, and this improvement is part of the v1.5.1 release.
